### ISSUE TITLE
Regent: Fill requires read privilege

### ISSUE THREAD

**Issue by manopapad** [StanfordLegion/legion#506]
The following program:

```
import 'regent'
task foo(r : region(double))
where writes(r) do
  fill(r, 0.0)
end
```

fails with:

```
/home/manolis/sw/legion/language/src/regent/type_check.t:2640: Errors reported during typechecking.
fill_priv.rg:4: invalid privileges in fill: reads($r)
  fill(r, 0.0)
     ^
```

**Comment by elliottslaughter** [comment]
Should be fixed in 6c68818845829424910938e848f2497c2cde59a8.

**Comment by manopapad** [comment]
Fixed
